src/java/org/apache/cassandra/metrics/TableMetrics.java

```diff
@@ -1143,23 +1098,29 @@ protected TableTimer createTableTimer(String name, Timer keyspaceTimer)
 
     protected TableTimer createTableTimer(String name, String alias, Timer keyspaceTimer)
     {
-        Timer cfTimer = Metrics.timer(factory.createMetricName(name), aliasFactory.createMetricName(alias));
-        register(name, alias, cfTimer);
-        return new TableTimer(cfTimer,
-                              keyspaceTimer,
-                              Metrics.timer(globalFactory.createMetricName(name),
-                                            globalAliasFactory.createMetricName(alias)));
+        return createTableTimer(name, alias, null, keyspaceTimer);
     }
 
-    protected Timer createTableTimer(String name)
+    protected TableTimer createTableTimer(String name, String alias, String deprecated, Timer keyspaceTimer)
```

Comment:
this method exists for a field which doesn't need `deprecated`, can we remove?

Reply:
Yup, I'll clean it up.